packages/terra-button/src/Button.module.scss

```diff
@@ -140,10 +145,16 @@
 
   .icon {
     display: inline-block;
+    line-height: 0;
   }
 
   .icon-svg {
     height: var(--terra-button-icon-height, 1rem);
     width: var(--terra-button-icon-width, 1rem);
+
+    svg {
+      //margin-top: 0.15em; // Helps with vertical centering as Icon vertical-align styles aren't applied properly in flexbox container
```

Comment:
Should probably be uncommented or removed

Reply:
Uncommented! - [ca5cb91](https://github.com/cerner/terra-core/pull/2378/commits/ca5cb9106a36879ce54010302eeb877b94e90b29)